lib/Widget/evaluationForms/post-hos/virtal_sign_form.dart

```diff
@@ -358,43 +389,36 @@ class _VirtalSignFormState extends State<VirtalSignForm> {
-                                            //         value >= 4)) {
-                                            //   showAdvise1(context, value,
-                                            //       _anSubCollection['state']);
-                                            //   if (checkNotificationCriteria(
-                                            //       value)) {
-                                            //     var creation =
-                                            //         _calculationService
-                                            //             .formatDate(
-                                            //                 date:
-                                            //                     DateTime.now());
-                                            //     var patientState =
-                                            //         _anSubCollection['state'];
-                                            //     _firebaseService
-                                            //         .addNotification({
-                                            //       'formName': 'pain',
-                                            //       'formId': formId,
-                                            //       'userId': UserStore
-                                            //           .getValueFromStore(
-                                            //               'storedUserId'),
-                                            //       'creation': creation,
-                                            //       'patientState': patientState,
-                                            //       'seen': false,
-                                            //     });
-                                            //   }
-                                            // }
+                                                    data: dataToDB,
+                                                    formTime: widget.formTime);
+                                            var patientState =
+                                                await _firebaseService
+                                                    .getPatientState(
+                                                        hn: widget.hn);
+                                            if ((patientState ==
+                                                    'Post-Operation@Hospital' &&
+                                                _pain != null)) {
+                                              if (checkNotificationCriteria(
+                                                      widget.hn, _pain) ||
+                                                  (_bt < 36.0 || _bt > 37.0) ||
+                                                  (_pr < 60.0 || _pr > 100) ||
+                                                  (_rr < 16 || _rr > 20) ||
+                                                  (_systolic > 130 ||
+                                                      _diastolic > 80)) {
```

Comment:
I think we should move these constants to the new file .
For exam. Const `btUpperCriteria = 37.0`  up to you in naming.

Reply:
Please check in fbfbbd3 that it is as recommended or not.